### ISSUE TITLE
Textarea doesn't have a focus outline in the docs

### ISSUE THREAD

**Issue by jhackett1** [LBHackney-IT/lbh-frontend#152]
https://design-system.hackney.gov.uk/components/textarea

not sure if it only affects the docs, or the underlying component itself.

should have a focus outline like: https://design-system.hackney.gov.uk/components/input
